### PR TITLE
Add deploy to Netlify section to docs

### PR DESCRIPTION
**Pull request by lunaceee**
We don't have instructions of how to deploy to Netlify in our docs.
I realized that we use plugins as default set up for deployment. 
I tried the [hexo-deployer-netlify](https://github.com/piktaszuikis/hexo-deployer-netlify) plugin but it doesn't quite work out of the box. 

I'm proposing adding a few lines to provide some instructions about deploying sites on Netlify for now so that it's visible to everyone.

### CODE REVIEW

**Comment by tcrowe** [comment]
Maybe also add method 3: push to github, gitlab, or whatever is supported and netlify should pick up the change automatically. Or web hooks? Maybe you can find out that method or others and that would help.

**Comment by tcrowe** [comment]
It grew! 😯

Are the images referencing the wrong path?

- [ ] fix img paths

**Comment by lunaceee** [comment]
It was not correct yes, my bad! Should be fixed now

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "version": "0.0.0",
   "private": true,
   "hexo": {
-    "version": "3.7.0"
+    "version": "3.7.1"
```
Comment (tcrowe) [inline package.json]: Usually we would want to keep the separate so it's topic-specific but that is the most recent version of hexo. 🤷🏼‍♂️

Comment (lunaceee) [reply package.json]: Ah...I didn't remember doing anything to it...let me change it back real quick. Thanks!

**Comment by tcrowe** [comment]
Thanks for entertaining my nitpick. :neckbeard:

File: source/docs/deployment.md [hunk 2]
```diff
@@ -70,6 +70,74 @@ Option | Description
...
+
+[Netlify](https://www.netlify.com/) provides continuous deployment services, global CDN, ultra-fast DNS, atomic deploys, instant cache invalidation, automatic HTTPS, a browser-based interface, a CLI, and many other features for managing your Hexo website. 
+
+Go to https://app.netlify.com/signup and select your preferred signup method. This will likely be a hosted Git provider, although you also have the option to sign up with an email address.
+
+There are two different ways to deploy your sites on Netlify.
+
+1. Use the web UI
+   
+   The following examples use GitHub, but other git providers will follow a similar process.
```
Comment (tcrowe) [inline source/docs/deployment.md]: Please unindent all Netlify lines so it fits to line start

File: source/docs/deployment.md [hunk 3]
```diff
@@ -70,6 +70,74 @@ Option | Description
...
 
+## Netlify
+
+[Netlify](https://www.netlify.com/) provides continuous deployment services, global CDN, ultra-fast DNS, atomic deploys, instant cache invalidation, automatic HTTPS, a browser-based interface, a CLI, and many other features for managing your Hexo website. 
+
+Go to https://app.netlify.com/signup and select your preferred signup method. This will likely be a hosted Git provider, although you also have the option to sign up with an email address.
+
+There are two different ways to deploy your sites on Netlify.
+
+1. Use the web UI
```
Comment (tcrowe) [inline source/docs/deployment.md]: An idea is to merge the `###` headings with the step number. e.g. `### 1. Create a Netlify account`

Comment (tcrowe) [reply source/docs/deployment.md]: Or eschew numbers is also good

**Comment by tcrowe** [comment]
Thank you @lunaceee for your contribution!

**Pull request merged**